lib/flipper/dsl.rb

```diff
@@ -34,6 +34,15 @@ def enabled?(name, *args)
       feature(name).enabled?(*args)
     end
 
+    def strict_enabled?(name, actor)
+      if enabled?(name)
```

Comment:
This still leaves open the possibility of enabled by percentage which could also be problematic 🤔